src/recs.js

```diff
@@ -215,6 +215,10 @@ export async function removeTemplate(options) {
 	}
 
 	const branchName = branch || repository.branch || DEFAULT_BRANCH;
+	if (!repository.branchList.includes(branchName)) {
```

Comment:
Might want some optional chaining here incase branchList is undefined (where it is defined is within a try/catch that does nothing).

Reply:
Not sure why but the optional chaining was not working, it was throwing errors at the ? characters. So I just did it the old fashion way. 
![B9727486-4E41-499A-9EA4-5D9890A38A9E_4_5005_c](https://user-images.githubusercontent.com/22894994/203426484-cbb26ffb-618d-462c-b5bc-4c8455c7b9e3.jpeg)

Reply:
Yea, I think we allowed for this in the `patch` branch (merged into Develop). Old fashioned is good for now :)